### PR TITLE
Add licenseHeader to gradle-spotless

### PR DESCRIPTION
**Pull request by kanterov**
It automatically adds license headers with `spotlessApply` and ensures
that there is no newline between license and package statement.

------------------------

Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

It will help us expedite review of your Pull Request if you tag someone (e.g. `@username`) to look at it.

Post-Commit Tests Status (on master branch)
------------------------------------------------------------------------------------------------

Lang | SDK | Apex | Dataflow | Flink | Gearpump | Samza | Spark
--- | --- | --- | --- | --- | --- | --- | ---
Go | [Build Status](https://builds.apache.org/job/beam_PostCommit_Go_GradleBuild/lastCompletedBuild/) | --- | --- | --- | --- | --- | ---
Java | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_GradleBuild/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Apex_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Dataflow_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Flink_Gradle/lastCompletedBuild/) [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_PVR_Flink/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Gearpump_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Samza_Gradle/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Java_ValidatesRunner_Spark_Gradle/lastCompletedBuild/)
Python | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_Verify/lastCompletedBuild/) | --- | [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_VR_Dataflow/lastCompletedBuild/) </br> [Build Status](https://builds.apache.org/job/beam_PostCommit_Py_ValCont/lastCompletedBuild/) | [Build Status](https://builds.apache.org/job/beam_PostCommit_Python_VR_Flink/lastCompletedBuild/) | --- | --- | ---

### CODE REVIEW

**Comment by kanterov** [comment]
retest please

**Comment by kanterov** [comment]
retest this please

File: runners/core-construction-java/src/main/java/org/apache/beam/runners/core/construction/graph/PipelineValidator.java [hunk 1]
```diff
@@ -1,17 +1,19 @@
 /*
- * Copyright (C) 2017 Google Inc.
```
Comment (kanterov) [inline runners/core-construction-java/src/main/java/org/apache/beam/runners/core/construction/graph/PipelineValidator.java]: @akedin can you double check this?

Comment (akedin) [reply runners/core-construction-java/src/main/java/org/apache/beam/runners/core/construction/graph/PipelineValidator.java]: Yup, it shouldn't be here :) Thank you!

**Comment by kanterov** [comment]
Run Java FnApi PreCommit

**Comment by kanterov** [comment]
@akedin seems everything is green, except two jobs that aren't configured yet

**Comment by akedin** [comment]
Merging, thank you for implementing this!

**Pull request merged**